options.go

```diff
@@ -280,6 +283,24 @@ func TombstonePeriod(period time.Duration) Option {
 	}
 }
 
+// SelfEvictPingDisable configures if ringpop should disable actively pinging
+// other members when it evicts itself from the network
```

Comment:
Could we improve the documentation here a bit by explaining when you would use this?